app/models/miq_server/worker_management/kubernetes.rb

```diff
@@ -255,6 +261,7 @@ def save_pod(pod)
     ch[:label_name]            = pod.metadata.labels.name
     ch[:last_state_terminated] = pod.status.containerStatuses.any? { |cs| cs.lastState.terminated }
     ch[:container_restarts]    = pod.status.containerStatuses.sum { |cs| cs.restartCount.to_i }
+    ch[:running]               = pod.status.phase == "Running" && pod.status.containerStatuses.all? { |cs| cs.ready && cs.started }
```

Comment:
The only change I see here is previously, we did a find on line 30 to get the first container status for the desired container name/deployment name. Then, we checked ready and started on that container.  Now, we do all containers and check they're ready and started.  I think that's fine but curious if that was intentional to fix something.

Reply:
Yes good point, it was intentional in the sense that the other checks here (last_state_terminated and container_restarts) check all containers in the pod so I thought it made sense to do the same for running, plus I didn't have the `worker.worker_deployment_name` as handy

Reply:
Awesome, thanks for confirming!  LGTM.